packages/contracts/contracts/interfaces/IOwnable.sol

```diff
@@ -0,0 +1,28 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.20;
+
+import { IPoll as IPollBase } from "maci-contracts/contracts/interfaces/IPoll.sol";
```

Comment:
Is this needed?

Reply:
I'd keep it consistent because poll has additional methods and it's not expected if platform poll doesn't have them.

Reply:
Oh, it's inside IOwnable, will remove it